### PR TITLE
Document typing information inside sdist

### PR DESCRIPTION
**Pull request by ssbarnea**
Documents how to assure that type information is includes with sdist files.

### Description

Following the current guidelines did not include type information in sdist, as they had effect only on built wheels.

### CODE REVIEW

File: docs/source/installed_packages.rst [hunk 1]
```diff
@@ -134,3 +134,12 @@ the ``setup.py`` might look like:
         package_data={"package_c-stubs": ["__init__.pyi", "lib.pyi"]},
         packages=["package_c-stubs"]
     )
+
+While instructions above assure that the builded wheel contains the type files,
+you may also need to modify your ``MANIFEST.in`` file to ensure their inclusion
+inside ``sdist`` (``.tag.gz``) archive too:
```
Comment (mgedmin) [inline docs/source/installed_packages.rst:140]: Typo:
```suggestion
inside ``sdist`` (``.tar.gz``) archive too:
```

File: docs/source/installed_packages.rst [hunk 2]
```diff
@@ -134,3 +134,12 @@ the ``setup.py`` might look like:
         package_data={"package_c-stubs": ["__init__.pyi", "lib.pyi"]},
         packages=["package_c-stubs"]
     )
+
+While instructions above assure that the builded wheel contains the type files,
+you may also need to modify your ``MANIFEST.in`` file to ensure their inclusion
+inside ``sdist`` (``.tar.gz``) archive too:
```
Comment (gvanrossum) [inline docs/source/installed_packages.rst:140]: Redundant word:
```suggestion
inside ``sdist`` (``.tar.gz``) archive:
```

**Review by gvanrossum — APPROVED** [review]

**Comment by JelleZijlstra** [comment]
Sorry this was never merged! I'll push the button as soon as CI passes again.

**Comment by JelleZijlstra** [comment]
Or I guess @hauntsaninja will since he's the one fixing the merge conflict, thanks!

**Pull request merged**